.env

```diff
@@ -0,0 +1,3 @@
+USER_EMAIL = smtp.gmail.com
+EMAIL_PASSWORD = agroeasy1#$##
+EMAIL_HOST = agroeasy2018@gmail.com
```

Comment:
@joshuaobinna Obinna why don't you understand that this file should not be committed to your branch. Remove this file from the commit